Source/Core/DolphinQt/GCMemcardManager.cpp

```diff
@@ -199,16 +212,11 @@ void GCMemcardManager::UpdateSlotTable(int slot)
 
     auto frames = GetIconFromSaveFile(file_index, slot);
```

Comment:
This could be renamed to `icon_data` for increased readability.

Reply:
Sure, might as well.